public/scss/components/Map.module.scss

```
@@ -10,17 +10,29 @@
 
 .mapSelectArea
 {
-  @extend .btn_transparent;
-  @include SmallSemiBold;
-  height: 103px;
+  @include flex(row, center, flex-start);
+  padding: 28px 18px 10px 18px !important;
   border: 1px solid $color_placeholder;
-  color: $color_text_secondary;
   background-color: $color_white;
-  
-  &:hover
+  width: 100%;
+}
+
+.mapSelectAreaSpan
+{
+  @include flex(row, center, center);
+  color: $color_text_grey;
+  @include RegulerReguler;

```

```suggestion
  @include flex(row, center, center);
  @include RegulerReguler;
  color: $color_text_grey;
```

include taro paling atas pacc